dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumRawEvent.kt

```diff
@@ -66,4 +66,6 @@ internal sealed class RumRawEvent {
     internal class SentError : RumRawEvent()
 
     internal class ViewTreeChanged : RumRawEvent()
+
+    internal class ResetSession : RumRawEvent()
```

Comment:
Is this only meant for tests ? If so...I am a bit reluctant....can't we just solve it with reflection somehow from the Shopist application ?

Reply:
It's already resolved through reflection in Shopist. The problem is I don't want to go too deep into the reflection. The deeper I go, the more fragile it will be. Any little change here might break the reflection in Shopist, inducing higher maintainance cost.
Having a single "high level" access allow us to update the implementation without having to update Shopist